# Define the goal
uint32 target_id						# ID of the target
string sender_UUID 						# ID of the CPS asking for cost computation
sensor_msgs/NavSatFix pose				# Position of the target
---
# Define the result
uint32 target_id						# ID of the target
sensor_msgs/NavSatFix pose				# Position of the target
---
# Define the feeeback